packages/nimble-components/src/combobox/index.ts

```diff
@@ -57,9 +58,33 @@ export class Combobox
     @attr({ attribute: 'error-visible', mode: 'boolean' })
     public errorVisible = false;
 
+    @observable
+    public region?: AnchoredRegion;
+
+    @observable
+    public controlWrapper?: HTMLElement;
+
     private valueUpdatedByInput = false;
     private valueBeforeTextUpdate?: string;
 
+    public regionChanged(
```

Comment:
These methods can be `protected` (or even `private`).

Reply:
Made them private